### ISSUE TITLE
can not configure integration

### ISSUE THREAD

**Issue by lecrams** [custom-components/ble_monitor#524]
After installing the integration it does not show up in HA configuration->integration.
The BLE dongle is detected by the OS (see below dmesg output)

[79901.913521] usb 1-1.3: new full-speed USB device number 6 using xhci-hcd
[79902.082677] cdc_acm 1-1.3:1.0: ttyACM1: USB ACM device
[79913.752986] usb 1-1.2: new full-speed USB device number 7 using xhci-hcd
[79913.925375] Bluetooth: hci0: CSR: Unbranded CSR clone detected; adding workarounds...

What can be wrong?

**Comment by Ernst79** [comment]
please check that the folder /custom_components/ble_monitor is present on your system (with content). make sure you restart HA, before trying to add the integration. Hit CTRL+F5 a couple of times if Passive BLE monitor is still not visible and try again.

**Comment by roberttco** [comment]
I manually configured the integration in the configuration.yaml file and it showed up in the integrations.  I then commented out the  entries in configuration.yml and I was then able to select the integration from the list of integrations. I made no changes to the file system permissions between these steps.  I did remove the integration after removing the values in the configuration.yaml file and I restarted Home Assistant.  I am using Home Assistant core.
